src/definitions.d.ts

```diff
@@ -3,8 +3,15 @@ import { BeamerConfig, BeamerMethods } from '@services/beamer/types'
 
 declare global {
   interface Window {
+    isDesktop?: boolean
+    ethereum?: {
+      autoRefreshOnNetworkChange: boolean
+      isMetaMask: boolean
+    }
```

Comment:
Were these ported over from `safe-react`? I know that `isDesktop` was our flag for the Electron app and the other two, I don't think we need as we never access the `window` object for them afaik.

I would say the same for `Cypress` below but I think that will be used for the injected test wallet.

Reply:
I think it's for the mock injected wallet.